src/main/java/com/fredsonchaves07/moviecatchapi/api/services/email/FakeSendMailService.java

```diff
@@ -18,12 +20,14 @@ public class FakeSendMailService implements SendEmailService {
     final Logger logger = LoggerFactory.getLogger(FakeSendMailService.class);
 
     @Override
-    public void send(MessageEmailDTO messageEmailDTO) throws SendEmailException {
+    public void send(
+            MessageEmailDTO messageEmailDTO, String template, HashMap<String, Object> templateParams
```

Comment:
Refactor to something like TemplateMailDTO to center the email template information in a specific class